db/user.go

```diff
@@ -221,3 +222,71 @@ func SyncUsers(course *model.Course, newUsers map[string]*model.User,
 
     return syncResult, nil;
 }
+
+// ResolveUsers is a helper function to maps string representations of roles and * (all roles) to the emails for users with those roles.
```

Comment:
Is it not a helper function, it's just a function.
There is no hard definition of a helper function, but this function is exported, which is definitely not true for helpers.